src/index.js

```diff
@@ -0,0 +1,99 @@
+import * as tk from './utility.js';
+import './style.css';
+
+const loadTask = document.querySelector('.todo-list');
+const form = document.querySelector('.todo-form');
+const clearList = document.querySelector('.clear');
+
+let taskIndex = 0;
+let taskDescription = '';
+const taskCompleted = '';
+
+const loadListElement = () => {
+  const li = document.createElement('li');
+  li.classList.add(
+    // 'list-group-item',
+    // 'justify-content-between',
+    // 'd-flex',
+    // 'align-items-center',
+    // eslint-disable-next-line comma-dangle
+    'task'
+  );
+  li.innerHTML = `<div class="form-check">
+  <input class="checkbox task-check" type="checkbox" ${taskCompleted}/>
+  <input class="desc todo-input" type="text" value= "${taskDescription}" id="${taskIndex}"/> 
+</div>`;
+
+  loadTask.append(li);
+};
+
+const displayBook = () => {
+  const tasks = tk.getLocalTasks();
+  tasks.forEach((task) => {
+    taskDescription = task.desc;
+    taskIndex = task.id;
+
+    loadListElement();
+  });
+};
+
+form.addEventListener('submit', (e) => {
+  const addDesc = document.querySelector('.todo-input').value;
+  e.preventDefault();
+  tk.addLocalTask(addDesc);
+});
+
+displayBook();
+
+const checkBox = document.querySelectorAll('.checkbox');
+checkBox.forEach((item) => {
+  item.addEventListener('click', () => {
+    const tasks = tk.getLocalTasks();
+    tasks.forEach((elem) => {
+      if (elem.id === parseInt(item.nextElementSibling.id, 10)) {
+        if (elem.completed === false) {
+          item.nextElementSibling.classList.add('completed');
+          elem.completed = true;
+        } else {
+          item.nextElementSibling.classList.remove('completed');
+          elem.completed = false;
+        }
+      }
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    });
+  });
+});
+
```

Comment:
- [ ] Kindly check your task completing function, whenever I mark a task completed the changes made are not stored in the local storage.